### PR TITLE
GPII-3987: Expose current system brightness in isolated function

### PR DESCRIPTION
**Pull request by JavierJF**

### CODE REVIEW

File: gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js [hunk 1]
```diff
@@ -415,3 +415,33 @@ gpii.deviceReporter.wmiSettingSupported = function (namespace, query) {
 
     return result;
 };
+
+/**
+ * Gets the current system brightness value making the proper
```
Comment (amb26) [inline gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js]: Please add a comment referencing GPII-3987 and also that this function will be removed once GPII-3958 is implemented

File: gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js [hunk 2]
```diff
@@ -415,3 +415,33 @@ gpii.deviceReporter.wmiSettingSupported = function (namespace, query) {
...
+ * query to the wmiSettingsHandler.
+ *
+ * @return {Object} The current system brightness, if the
```
Comment (amb26) [inline gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js]: Is it an Object? It looks like it is a Number.... please also report what the scale of the value is

File: gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js (hunk 1 above)
Comment (JavierJF) [reply gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js]: Done!

File: gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js (hunk 2 above)
Comment (JavierJF) [reply gpii/node_modules/wmiSettingsHandler/src/WmiSettingsHandler.js]: Yes, both observation were right, already done!

**Comment by amb26** [comment]
Sorry to leave the review comments - I understood from our updates on Monday that this work would now no longer be required, since the next built of Morphic will not include the Brightness setting, and by the time we make the next build, @cindyli 's work on https://issues.gpii.net/browse/GPII-3958 will be complete.
Let me know if this memory/understanding of our status on Monday agrees with yours - if so, we can close this pull and the associated JIRA

**Comment by JavierJF** [comment]
@amb26 yes, that was my understanding too, I just completed the feedback, just in case this works became relevant again before we have the other solution in place. It was just a 'anticipatory movement'. Now that it's addressed we can close everything since as you said, it looks like it wont be needed!

**Pull request closed without merging**